test/e2e/providers/cloudprovider.go

```diff
@@ -39,14 +40,22 @@ func NewCloudProvider() (CloudProvider, error) {
 	}
 	switch provider := infra.Status.PlatformStatus.Type; provider {
 	case config.AWSPlatformType:
-		return awsProvider.New(openshift, &infra.Status)
+		return awsProvider.New(openshift, &infra.Status, windowsServerVersion)
 	case config.AzurePlatformType:
-		return azureProvider.New(openshift, &infra.Status), nil
+		return azureProvider.New(openshift, &infra.Status, windowsServerVersion), nil
 	case config.GCPPlatformType:
-		return gcpProvider.New(openshift, &infra.Status), nil
+		return gcpProvider.New(openshift, &infra.Status, windowsServerVersion), nil
 	case config.VSpherePlatformType:
+		if windowsServerVersion != windows.Server2022 {
+			return nil, fmt.Errorf("%v provider does not support Windows Server %s", provider,
+				windowsServerVersion)
+		}
 		return vSphereProvider.New(openshift, &infra.Status)
 	case config.NonePlatformType:
+		if windowsServerVersion != windows.Server2022 {
+			return nil, fmt.Errorf("%v provider does not support Windows Server %s", provider,
```

Comment:
I thought we supported 2019 on platform-none, at least [according to our docs](https://docs.openshift.com/container-platform/4.12/windows_containers/understanding-windows-container-workloads.html#wmco-prerequisites-supported-6.0.0_understanding-windows-container-workloads). Or is this done because of how the golden image template is hardcoded in the release repo steps?

Reply:
This change is no longer needed.